themes/default/content/blog/google-cloud-7-0/index.md

```diff
@@ -0,0 +1,211 @@
+---
+title: "Pulumi Google Cloud Classic 7.0"
+date: 2023-11-05T09:00:00-04:00
+draft: false
+meta_desc: "The Latest for Your Modern Cloud Infrastructure: Pulumi Google Cloud Provider 7.0."
+meta_image: meta.png
+authors:
+  - monica-rodriguez
+tags:
+  - google-cloud
+  - releases
+
+---
+
+The latest major release of the Pulumi Google Cloud Classic Provider is available now! This [7.0 release](https://github.com/pulumi/pulumi-gcp/releases/tag/v7.0.0) contains the latest upstream changes to keep you up-to-date along with a highly requested bug fix, keeping your journey in managing Google Cloud resources fresh and smooth.
+
+The Pulumi Google Cloud Classic provider can be used to provision any of the cloud resources available in the upstream provider. It is part of the suite of [Pulumi official providers](https://www.pulumi.com/docs/intro/cloud-providers/), which means that it is officially maintained and supported by Pulumi. The provider is also open source and available on [GitHub](https://github.com/pulumi/pulumi-gcp) for you to contribute to and improve.
+
```

Comment:
Perhaps we can add a CTA here which says "If you want to use GCP with Pulumi right now, you can use one of these templates to get started fast."

Reply:
Updated per feedback. @aaronkao Please take another look when you get a chance. Thanks!